event/views.py

```diff
@@ -659,8 +666,9 @@ class EventGet(generics.GenericAPIView):
     def get(self, request, *args, **kwargs):
         try:
             event = Event.objects.get(pk=kwargs["pk"])
-            serialized_event = self.get_serializer(event)
-            return Response(serialized_event.data, status=status.HTTP_200_OK)
+            serialized_event = EventSerializer(event, context={"user": request.user})
+            completed_data = serialized_event.data.copy()
+            return Response(completed_data, status=status.HTTP_200_OK)
```

Comment:
```suggestion
            serialized_event = EventSerializer(event, context={"user": request.user})
            return Response(serialized_event.data, status=status.HTTP_200_OK)
```

Por qué hace falta la copia?

Reply:
Se coló, arreglado.